spec/features/visitor_can_view_items_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+feature "View items" do
+  let!(:item_one) {Item.create(name: "Canon 7d",
+                                description: "The best camera ever (NOT)",
+                                price: 15)}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 6, 43 should be on its own line.